cumulus_library_data_metrics/c_resources_per_pt/c_resources_per_pt.jinja

```diff
@@ -72,7 +72,19 @@ SELECT
     MAX(num_resources) AS "max"
 FROM summed_counts
 UNION
+-- duplicate the above for consistency with flagging
+-- "no category available" situations via a NULL category
+SELECT
+    'cumulus__all' AS resource,
+    CAST(NULL AS VARCHAR) AS category,
+    CAST(AVG(num_resources) AS DECIMAL(18, 2)) AS average,
+    CAST(STDDEV_POP(num_resources) AS DECIMAL(18, 2)) AS std_dev,
```

Comment:
this is an aside, but we should probably make sure the dashboard renders this with the same level of precision as this.